meta-iot2000-bsp/recipes-kernel/linux/files/0007-serial-8250_pci-Add-support-for-accessing-red-LED-vi.patch

```diff
@@ -0,0 +1,127 @@
+From 403bfe046b658799a8735cf890da21f8e42280bc Mon Sep 17 00:00:00 2001
+From: Frank Ehlis <frank.ehlis@siemens.com>
+Date: Thu, 29 Sep 2016 14:48:52 +0200
+Subject: [PATCH] serial: 8250_pci: Add support for accessing red LED via ledfs
+
+The LED-init was moved past pci_default_setup() so that
+port.membase is valid when saving it to our private data.
```

Comment:
That is not the right kind of commit log. It should describe where this LED comes from and why it is added to a UART driver.

My feeling is that this is not the best place. Maybe there is a smarter way to make the MPIOs available to drivers outside of the UART, but the given approach is probably the best we can do for now. Will become clearer once we submit these to upstream...